doc/stdlib/collections.rst

```diff
@@ -727,71 +1211,129 @@ Set-Like Collections
 
 .. function:: Set([itr])
 
-   Construct a :obj:`Set` of the values generated by the given iterable object, or an empty set.
-   Should be used instead of :obj:`IntSet` for sparse integer sets, or for sets of arbitrary objects.
+   Construct a "Set" of the values generated by the given iterable object, or an empty set. Should be used instead of "IntSet" for sparse integer sets, or for sets of arbitrary objects.
```

Comment:
Did we just throw hundreds of cross-references away? Does anyone have idea of just how many man-hours of cross-referencing went into these things???!!!!